spec/models/board_spec.rb

```diff
@@ -2,11 +2,47 @@
 require 'matrix'
 
 RSpec.describe Board, type: :model do
-  describe 'new board' do
-    let(:matrix) { Matrix.build(3) { '_'} }
-    context do
-      b = Board.new
+  let(:matrix) { Matrix.build(3) { '_'} }
```

Comment:
Space missing inside }.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.